### PR TITLE
Add small badge size

### PR DESCRIPTION
**Pull request by ksmort**
## Description
Add size prop for badge.  Size prop has `small` and `medium`, which can be used when the `variant` prop is set to `standard`.
Add example.

## Screenshots
Screenshot_2020-10-15 Badge Riipen UI

## Where to Start
components/Badge

### CODE REVIEW

File: packages/riipen-ui-docs/src/pages/components/badge/badge.md [hunk 1]
```diff
@@ -43,3 +43,9 @@ You can use the `anchor` property to move the badge to any corner of the wrapped
 You can use the `color` property choose the badge color.
 
 {{"demo": "pages/components/badge/Color.js"}}
+
+## Size
+
+You can use the `size` property to choose the badge size, if the `variant` is `standard`.
```
Comment (ksmort) [inline packages/riipen-ui-docs/src/pages/components/badge/badge.md:49]: Maybe double-check that this sounds nice.

File: packages/riipen-ui/src/components/Badge.jsx [hunk 2]
```diff
@@ -149,14 +159,25 @@ class Badge extends React.Component {
...
             font-family: ${theme.typography.fontFamily};
-            font-size: 12px;
             font-weight: ${theme.typography.fontWeight.medium};
-            height: ${RADIUS_STANDARD * 2}px;
             justify-content: center;
+            position: absolute;
+          }
+
+          .badge.small {
+            font-size: ${theme.typography.h5.fontSize};
```
Comment (ksmort) [inline packages/riipen-ui/src/components/Badge.jsx:168]: 10px

File: packages/riipen-ui-docs/src/pages/components/badge/badge.md (hunk 1 above)
Comment (stredger) [reply packages/riipen-ui-docs/src/pages/components/badge/badge.md:49]: ah yes my language skills are well known.
image

Comment (stredger) [reply packages/riipen-ui-docs/src/pages/components/badge/badge.md:49]: Sounds ok maybe something like?
```
With the `standard` `variant`, use the `size` property to set the badge size.
```
Or
```
Use the `size` property to choose the badge size. Only works with the `standard` `variant`
```
If you dont like those keeping as works as well

File: packages/riipen-ui/src/components/Badge.jsx [hunk 3]
```diff
@@ -99,10 +105,13 @@ class Badge extends React.Component {
       max,
       overlap,
       showZero,
+      size,
       variant
     } = this.props;
 
-    if ((content === 0 && !showZero) || (content == null && variant !== "dot"))
+    const isStandardVariant = variant !== "dot";
```
Comment (stredger) [inline packages/riipen-ui/src/components/Badge.jsx:112]: 👍

**Review by stredger — APPROVED** [review]
Cool beans

**Pull request merged**